src/components/checkboxes/index.md.njk

```diff
@@ -79,6 +79,18 @@ Keep it simple - if you need to add a lot of content, consider showing it on the
 
 Users are not always notified when conditionally revealed content is expanded or collapsed. This fails [WCAG 2.1 success criterion 4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html). We plan to fix this issue by the end of March 2021.
 
+### Including a ‘None of the above’ option
+
+Sometimes it is better to allow users to explicitly say that none of the options apply, rather than leaving them all unchecked. This can also help prevent users unintentionally not answering the question.
+
+The “None” option should appear last in the list, and be separated from the other options using an “or” divider.
+
+Adding the `exclusive` behaviour to the "None" checkbox enables some javascript which unselects all other checkboxes when checking this box.
+
+{{ example({group: "components", item: "checkboxes", example: "with-none-option", html: true, nunjucks: true, open: false, size: "xl"}) }}
+
+If javascript is unavailable, and a user selects both the ‘None’ checkbox and another checkbox, display an error message (see below).
```

Comment:
I'd also suggest moving this new heading just after the 'checkbox items with hints' heading, as it'd be good to keep the 'conditionally revealing content' heading as close to 'research' as possible.

```suggestion
### Add an option for ‘none of the above’

In some cases, it can be helpful to let users select an checkbox to say that none of the options apply to them, instead of leaving them all unchecked. This can also help make sure users do not skip the question by accident.

Show the option for “none” checkbox option last. Separate it from the other options using with an “or” divider.

Add the `exclusive` behaviour to the "none" checkbox to enable some javascript that unselects all other checkboxes when the user clicks "none".

{{ example({group: "components", item: "checkboxes", example: "with-none-option", html: true, nunjucks: true, open: false, size: "xl"}) }}

If javascript is unavailable, and a user selects both the ‘None’ checkbox and another checkbox, display an error message (see below).
```